go.mod

```diff
@@ -1,44 +1,42 @@
 module github.com/db-operator/db-operator
 
-go 1.22.0
-
-toolchain go1.22.4
+go 1.23.1
```

Comment:
It's supposed to happen here: https://github.com/db-operator/db-operator/pull/67

So maybe we can wait a bit?